include/circt/Dialect/Calyx/CalyxPrimitives.td

```diff
@@ -378,6 +378,40 @@ def AddFNOp : ArithBinaryFloatingPointLibraryOp<"addFN"> {
   }];
 }
 
+def MulFNOp : ArithBinaryFloatingPointLibraryOp<"mulFN"> {
```

Comment:
```suggestion
def IEEEMulFNOp : ArithBinaryIEEEFloatingPointLibraryOp<"ieeeMulFN"> {
```

Are you imagining something like this? @rachitnigam

Reply:
No I think @rachitnigam is imagining changing the op name to `ieee754.mul`, which is totally allowed by MLIR.